update agorachat ios getstart doc (#2477)
AgoraIO/agora_doc_source: en-US/markdown/agora-chat/Getting Started/agora_chat_get_started_ios.md

        self.view.addSubview(sendButton)
        logView = createLogView()
        self.view.addSubview(logView)

>         // Input userId and token which generated on console

Reviewer: ```suggestion
        // Enter the user ID and token generated on Agora Console
```

> 

Reviewer: 第 152 行，如果是生成一条运行日志改为：Outputs a running log

若是多条，用 Outputs running logs

        options.enableConsoleLog = true
        AgoraChatClient.shared.initializeSDK(with: options)
        // Adds the chat delegate to receive messages

>         AgoraChatClient.shared.chatManager?.add(self, delegateQueue: nil)

Reviewer: 第 196 行，改为：Calls the SDK initialization function

// Button action
extension ViewController {

>     // login with the token.

Reviewer: 第 203 行，建议：Creates an account and log in with the token

Use Xcode to compile and run the project on an iOS device or an emulator.
To try sending and receiving text messages, follow the steps on two clients:

> 1. Log into Agora Chat as `lxm`, type `lxm2` for `Peer username`, and send a text message.

Reviewer: 建议，将 log into 全部替换为 log in to，统一表达。

  ![](../images/getStart/iOS/send.png)

2. Log into Agora Chat as `lxm2` and check logs to see whether this message is received.

>   ![](../images/getStart/iOS/receive.png)

Reviewer: 第 294 行的`AgoraChatExample`  改为 'Your project target'
